test/test-config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "environments": {
+    "dev": {
+      "account": "222222222222",
+      "region": "us-east-1",
+      "resources": {
+        "AWS::Lambda::Function": {
```

Comment:
So the current identifier used is the resource type? Meaning all Lambda functions would be impacted by the Aspect? Wouldn't that be too broad?

Reply:
Yea, we should be able to implement by construct `id` as well. I can add that support in a future PR.

Reply:
Will add in this PR.